### DISCUSSION TITLE
Android client

### DISCUSSION THREAD

**Discussion by teobaranga** [monicahq/monica#6840] [Ideas]
Hi, I recently discovered this project and I've really enjoyed using it so far. I would like to use it more on mobile but the website isn't the most mobile-friendly, especially the journal side. I don't think there is a mobile app available either which brings me to the question: is there any interest for a Monica mobile app? As an Android dev, I'd like to build one and it would be even nicer if other people are interested ✌️

**Comment by Xyz00777** [comment]
Would also like to see an mobile endpoint client, it would be an good idea to programm that based on flutter so they dont need to build separately for the different Operating Systems.
https://flutter.dev/multi-platform/mobile

**Reply by teobaranga** [reply]
I'm afraid I don't know any Flutter so I can only work on an Android app for now ✌️ Nothing stops others from building a Flutter app though

**Reply by Xyz00777** [reply]
is there already an beta version available i cant find an repository in monicahq

**Reply by Xyz00777** [reply]
i dont want to overhelm your programming :)
it looks like ios is in beta for kotlin and it looks like a desktop application is also possible based on kotlin :O
but your choice :)
either way its cool what your doing :+1: 
https://medium.com/ekino-france/building-your-first-desktop-application-in-kotlin-with-compose-desktop-ff8a9191ee34
https://www.reddit.com/r/Kotlin/comments/14hc6v0/comment/jpbgwnm/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button
